Services/src/main/java/com/agilepro/persistence/entity/admin/UserSettingEntity.java

```diff
@@ -0,0 +1,77 @@
+package com.agilepro.persistence.entity.admin;
+
+import javax.persistence.Column;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+import com.agilepro.commons.models.customer.UserSettingModel;
+import com.yukthi.persistence.annotations.UniqueConstraint;
+import com.yukthi.persistence.annotations.UniqueConstraints;
+import com.yukthi.utils.annotations.PropertyMapping;
+import com.yukthi.webutils.repository.UserEntity;
+import com.yukthi.webutils.repository.WebutilsEntity;
+
+/**
+ * The Class UserSettingEntity.
+ * 
+ * @author Pritam
+ */
+@Table(name = "USER_SETTING")
+@UniqueConstraints({ @UniqueConstraint(name = "SPACE_ID_NAME", fields = { "spaceIdentity", "userEntity", "projectEntity"}) })
+public class UserSettingEntity extends WebutilsEntity
+{
+	/** 
+	 * The user entity. 
+	 **/
+	@OneToOne
+	@Column(name = "USER_ID", nullable = false)
+	@PropertyMapping(type = UserSettingModel.class, from = "userId", subproperty = "id")
+	private UserEntity userEntity;
+	
+	/** 
+	 * The project entity. 
+	 **/
+	@OneToOne
+	@Column(name = "ACTIVE_PROJECT_ID", nullable = false)
+	@PropertyMapping(type = UserSettingModel.class, from = "projectId", subproperty = "id")
+	private ProjectEntity projectEntity;

```

Comment:
Project entity should not be there as part of user settings.

User settings should have - User id, Key (string), value (string of 2000 len).

And you can use it for any prupose like active project